File: CodeWars/CodeGarden/19-SumOfSequence.cs
public static class Kata
{
  public static int SequenceSum(int start, int end, int step)
  {
    if(start > end) { return 0; }

    var sum = 0;

    var currentValue = start;

    while(currentValue <= end)
    {
        sum += currentValue;
        currentValue += step;
    }

    return sum;
  }
}

File: CodeWars/CodeGarden/19-SumOfSequence.js
const sequenceSum = (begin, end, step) => {

    if(begin > end) { return 0; }

    let sum = 0;

    let currentValue = begin;

    while(currentValue <= end)
    {
        sum += currentValue;
        currentValue += step;
    }

    return sum;

  };


  sequenceSum(2, 6, 2);
  sequenceSum(1, 5, 1)
  sequenceSum(1, 5, 3)